src/Clap-Core.package/AliasNotFoundException.class/README.md

```diff
@@ -0,0 +1 @@
+I am AliasNotFoundException, an Error signaled when you try to remove an alias which doesn't exist in a flag
```

Comment:
removing doesn't really make sense… unless you have a specific use case in mind?